static/app/components/alerts/notificationBar.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+
+import {IconInfo} from 'sentry/icons';
+import space from 'sentry/styles/space';
+
+export const StyledNotificationBarIconInfo = styled(IconInfo)`
+  margin-right: ${space(1)};
+  color: ${p => p.theme.alert.info.iconColor};
+`;
+
+export const NotificationBar = styled('div')`
```

Comment:
Another new notification bar? @vuluongj20 do we have anything reusable for this

Reply:
@evanpurkhiser @vuluongj20 My plan is to make this the re-usable component and use this in getsentry

Reply:
Can you make a diff PR and create a story for it?

Reply:
no

Reply:
lol

Reply:
have I reached the limit to my power???

Reply:
Your reign of terror is over Evan, now my reign of terror begins 😈

Reply:
but ok i'll add it in a follow-up PR, I promise